src/components/ThemeSwitch.tsx

```diff
@@ -9,6 +10,12 @@ const ThemeSwitch: FC = () => {
   const dispatch = useDispatch()
   const light = useSelector(state => theme(state) === 'Light')
 
+  if (light) {
+    WebviewBackground.changeBackgroundColor({ color: '#ffffff' })
```

Comment:
Please use the PandaCSS design token for `color.bg` rather than hardcoding hex colors.